docs/content/preview/yugabyte-platform/create-deployments/create-universe-multi-zone-kubernetes.md

```diff
@@ -43,42 +43,47 @@ If no universes have been created yet, the **Dashboard** does not display any.
 - In the **Volume Info** field, specify the number of volumes multiplied by size. The default is 1 x 100GB.
-- Select **Enable YSQL** to enable the YSQL API endpoint to run Postgres-compatible workloads. This setting is enabled by default.
-- Select **Enable YEDIS** to enable the YEDIS API endpoint to run Redis-compatible workloads. This setting is disabled by default.
-- Select **Enable Node-to-Node TLS** to enable encryption-in-transit for communication between the database servers. This setting is enabled by default.
-- Select **Enable Client-to-Node TLS** to enable encryption-in-transit for communication between clients and the database servers. This setting is enabled by default.
-- Select **Enable Encryption at Rest** to enable encryption for data stored on the tablet servers. This setting is disabled by default.
+- Use the **Root Certificate** field to select an existing security certificate or create a new one.
+- Use the **Enable YSQL** field to specify whether or not to enable the YSQL API endpoint for running PostgreSQL-compatible workloads. This setting is enabled by default.
+- Use the **Enable YSQL Auth** field to specify whether or not to enable the YSQL password authentication.
+- Use the **Enable YCQL** field to specify whether or not to enable the YCQL API endpoint for running Cassandra-compatible workloads. This setting is enabled by default.
+- Use the **Enable YCQL Auth** field to specify whether or not to enable the YCQL password authentication.
+- Use the **Enable YEDIS** field to specify whether or not to enable the YEDIS API endpoint for running Redis-compatible workloads. This setting is disabled by default.
+- Use the **Enable Node-to-Node TLS** field to specify whether or not to enable encryption-in-transit for communication between the database servers. This setting is enabled by default.
+- Use the **Enable Client-to-Node TLS** field to specify whether or not to enable encryption-in-transit for communication between clients and the database servers. This setting is enabled by default.
+- Use the **Enable Encryption at Rest** field to specify whether or not to enable encryption for data stored on the tablet servers. This setting is disabled by default.
 
 Complete the **Advanced** section as follows:
 
 - In the **DB Version** field, specify the YugabyteDB version. The default is either the same as the YugabyteDB Anywhere version or the latest YugabyteDB version available for YugabyteDB Anywhere.
+- Use the **Enable IPV6** field to specify whether or not you want to use IPV6 networking for connections between database servers. This setting is disabled by default.
+- Use the **Enable Public Network Access** field to specify whether or not to assign a load balancer or nodeport for connecting to the database endpoints over the internet. This setting is disabled by default.
+- Use the **Enable Systemd Services** field to specify whether or not systemd services are allowed to run. This setting is enabled by default.
```

Comment:
I guess **Enable Systemd Services**  is not available for kubernetes universes  @iSignal @lizayugabyte